### PR TITLE
changed copy and comment

### PR DESCRIPTION
**Pull request by Kvaz1r**
Added the ability to copy and commenting for multiple selected areas.

### CODE REVIEW

**Comment by nelsam** [comment]
I'll review this as soon as I get my working directory to a better state.  Sorry for the delay.

File: commands/copy_paste.go [hunk 1]
```diff
@@ -37,11 +38,15 @@ func (c *Copy) Exec(target interface{}) (executed, consume bool) {
...
 		return true, true
 	}
-	if editor.Controller().SelectionCount() != 1 {
-		log.Print("Warning: copy can currently only copy the first selection")
+
+	selections := editor.Controller().Selections()
+	var buffer bytes.Buffer
+	for i := 0; i < selections.Len(); i++ {
+		buffer.WriteString(editor.Controller().SelectionText(i))
+		buffer.WriteString("\n")
```
Comment (nelsam) [inline commands/copy_paste.go]: This is going to add a newline to any copied text (even if you're only copying a single selection).  I think this needs to change in one of the following ways:

A) Remove this line and assume that multiple selected blocks of text should be concatenated
or
B) Skip this line if `i == selections.Len() - 1`

I think I would prefer option (A), personally.  If I'm selecting multiple things and copying them, I think I would prefer to not have a newline between each of my selections.

**Comment by nelsam** [comment]
I would like to see the `copy` command updated before I merge this.  The change to the `add-comments` command looks good, though.

File: commands/copy_paste.go (hunk 1 above)
Comment (Kvaz1r) [inline commands/copy_paste.go]: Good, I removed the string.

**Pull request merged**